src/style.css

```diff
@@ -0,0 +1,94 @@
+@font-face {
+  font-family: "Avenir";
+  src: url(./fonts/Avenir-Book.ttf);
+  font-style: normal;
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: "Avenir";
+  src: url(./fonts/Avenir-Heavy.ttf);
+  font-style: normal;
+  font-weight: 800;
+}
+
+body {
+  font-family: "Avenir", sans-serif;
+  font-weight: 300;
```

Comment:
move it to the `html` tag selector

Reply:
move this into `html` and add `font-family: inherit;` in `.form-field`